index.html

```diff
@@ -44,8 +44,6 @@ <h2>Would you recommend freeCodeCamp to a friend?</h2>
         <label for="Maybe">Maybe</label><br>
         <input type="radio" value="Not sure" id="NoSure">
         <label for="NoteSure">Note sure</label>
-
-
         <label for="">Front-end Projects</label>
         <input type="checkbox">
```

Comment:
### Be careful with blank lines and indentation.

1. Do not add blank lines, spaces, or indentations without a reason.
2. For readability, add blank lines to separate large or logical code blocks.
3. For readability, add two spaces of indentation. Do not use the tab key.
4. on some code editors, you can set the tab button to do an X amount of spaces.